build/tasks/CreateFrameworkListFile.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
This isn't our usual C# license header and it's aligned with neither `<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>` in the root Directory.Build.props file nor the content of our LICENSE.txt file.

Reply:
This file was copied from Core-setup. I'll change the header.